### DISCUSSION TITLE
404:NOT_FOUND locally working app not working in deploy, vercel-given url, ReactJS

### DISCUSSION THREAD

**Discussion by Sagr859** [vercel/community#3800] [Help] [answered]
### Summary

I have deployed both frontend (ReactJS) and backend (NodeJS, Express,MongoDB) in vercel. The backend is working fine, it returns responses in browser and in local frontend app. I tried to deploy the frontend via GitHub, the deployed site says : 
file:///home/sagar/Pictures/Screenshots/Screenshot%20from%202023-08-28%2013-26-59.png

### Example

https://sagarstockmockfrnt-iq4tvaz1j-sagr859.vercel.app/

### Steps to Reproduce

https://sagarstockmockfrnt-iq4tvaz1j-sagr859.vercel.app/

**Comment by amyegan** [comment]
Hey @Sagr859. If you share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give me a better view of what is going wrong.

**Reply by Sagr859** [reply, marked as answer]
Oh, I'm sorry guys. Turns out its a blunder from my side.
I added the 'vercel.json' of backend in this directory too. So I believe Vercel tried to use it and failed to run REactJS properly.

Sorry for the inconvenience caused... 🙏
